### ISSUE TITLE
Got bad results from install scripts & Access is denied. but could connect the remote host in command line

### ISSUE THREAD

**Issue by yanhui09** [microsoft/vscode-remote-release#5803]
- VSCode Version: 1.61.2
- Local OS Version: windows 10 21H1
- Remote OS Version: ubuntu 18.04
- Remote Extension/Connection Type: SSH
- Logs:

```
[21:52:47.449] Log Level: 2
[21:52:47.459] remote-ssh@0.65.8
[21:52:47.459] win32 x64
[21:52:47.461] SSH Resolver called for "ssh-remote+10.61.80.76", attempt 1
[21:52:47.462] "remote.SSH.useLocalServer": false
[21:52:47.462] "remote.SSH.showLoginTerminal": false
[21:52:47.463] "remote.SSH.remotePlatform": {"yan":"linux","yanhui":"linux"}
[21:52:47.463] "remote.SSH.path": undefined
[21:52:47.463] "remote.SSH.configFile": undefined
[21:52:47.464] "remote.SSH.useFlock": true
[21:52:47.464] "remote.SSH.lockfilesInTmp": false
[21:52:47.465] "remote.SSH.localServerDownload": auto
[21:52:47.465] "remote.SSH.remoteServerListenOnSocket": true
[21:52:47.466] "remote.SSH.showLoginTerminal": false
[21:52:47.469] "remote.SSH.defaultExtensions": []
[21:52:47.470] "remote.SSH.loglevel": 2
[21:52:47.470] SSH Resolver called for host: 10.61.80.76
[21:52:47.471] Setting up SSH remote "10.61.80.76"
[21:52:47.509] Using commit id "6cba118ac49a1b88332f312a8f67186f7f3c1643" and quality "stable" for server
[21:52:47.513] Install and start server if needed
[21:52:50.189] Checking ssh with "ssh -V"
[21:52:50.271] > OpenSSH_for_Windows_8.1p1, Libre
[21:52:50.271] > SSL 3.0.2

[21:52:50.277] Running script with connection command: ssh -T -D 1142 "10.61.80.76" bash
[21:52:50.280] Terminal shell path: C:\WINDOWS\System32\cmd.exe
[21:52:50.625] > ]0;C:\WINDOWS\System32\cmd.exe
[21:52:50.626] Got some output, clearing connection timeout
[21:52:50.653] > Access is denied.
[21:52:50.734] > The authenticity of host '10.61.80.76 (10.61.80.76)' can't be established.  
> ECDSA key fingerprint is SHA256:s8oA67K5GxAwgB4Cit7o0jgGgKhWosXjsBTNuSR80H0.
> Are you sure you want to continue connecting (yes/no/[fingerprint])?        
[21:52:50.734] Detected fingerprint confirmation message
[21:52:50.734] Showing fingerprint confirmation dialog
[21:53:01.728] Got fingerprint response: yes
[21:53:01.729] "install" wrote data to terminal: "yes"
[21:53:01.748] > y
[21:53:01.764] > Are you sure you want to continue connecting (yes/no/[fingerprint])? yes    
> Warning: Permanently added '10.61.80.76' (ECDSA) to the list of known hosts.
[21:53:01.926] > yanhui@10.61.80.76's password: 
[21:53:01.927] Showing password prompt
[21:53:05.778] Got password response
[21:53:05.779] "install" wrote data to terminal: "********"
[21:53:05.793] > 
[21:53:07.386] "install" terminal command done
[21:53:07.386] Install terminal quit with output: 
[21:53:07.387] Resolver error: Error: Got bad result from install script
	at Function.Create (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:397041)
	at Object.t.tryInstall (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:489077)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:453110
	at async Object.t.withShowDetailsEvent (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:456460)
	at async Object.t.resolve (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:454189)
	at async c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:526942
[21:53:07.392] ------
```

Steps to Reproduce:

1. I have been using the vscode to connect my host within our university network (through VPN). Today I couldn't connect to it anymore through vscode. The previous ssh config (using RSA-keys) couldn't work, but telling me `Access is denied` . I tried to remove the .vscode-server directory and rebuild .ssh/config on my local host to only use password. But still I couln't connect to the remote host, leaving me a slight different message --- `Got bad results from install scripts`.
However, I couldn't connet the remote host using ssh in powershell.

Below is the log using my previous RSA ssh config.

```
[21:47:17.220] Log Level: 2
[21:47:17.228] remote-ssh@0.65.8
[21:47:17.228] win32 x64
[21:47:17.229] SSH Resolver called for "ssh-remote+yan", attempt 1
[21:47:17.229] "remote.SSH.useLocalServer": false
[21:47:17.229] "remote.SSH.showLoginTerminal": false
[21:47:17.229] "remote.SSH.remotePlatform": {"yan":"linux","yanhui":"linux"}
[21:47:17.230] "remote.SSH.path": undefined
[21:47:17.230] "remote.SSH.configFile": undefined
[21:47:17.230] "remote.SSH.useFlock": true
[21:47:17.230] "remote.SSH.lockfilesInTmp": false
[21:47:17.230] "remote.SSH.localServerDownload": auto
[21:47:17.230] "remote.SSH.remoteServerListenOnSocket": true
[21:47:17.230] "remote.SSH.showLoginTerminal": false
[21:47:17.230] "remote.SSH.defaultExtensions": []
[21:47:17.230] "remote.SSH.loglevel": 2
[21:47:17.231] SSH Resolver called for host: yan
[21:47:17.231] Setting up SSH remote "yan"
[21:47:17.258] Using commit id "6cba118ac49a1b88332f312a8f67186f7f3c1643" and quality "stable" for server
[21:47:17.262] Install and start server if needed
[21:47:17.268] Checking ssh with "ssh -V"
[21:47:17.402] > OpenSSH_for_Win
[21:47:17.403] > dows_8.1p1, LibreSSL 3.0.2

[21:47:17.409] Running script with connection command: ssh -T -D 9066 yan bash
[21:47:17.413] Terminal shell path: C:\WINDOWS\System32\cmd.exe
[21:47:17.818] > ]0;C:\WINDOWS\System32\cmd.exe
[21:47:17.818] Got some output, clearing connection timeout
[21:47:17.860] > Access is denied.
[21:47:19.683] "install" terminal command done
[21:47:19.684] Install terminal quit with output: ]0;C:\WINDOWS\System32\cmd.exeAccess is denied.
[21:47:19.684] Received install output: ]0;C:\WINDOWS\System32\cmd.exeAccess is denied.
[21:47:19.686] Failed to parse remote port from server output
[21:47:19.687] Resolver error: Error: 
	at Function.Create (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:397041)
	at Object.t.handleInstallOutput (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:395684)
	at Object.t.tryInstall (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:489314)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:453110
	at async Object.t.withShowDetailsEvent (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:456460)
	at async Object.t.resolve (c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:454189)
	at async c:\Users\h3039\.vscode\extensions\ms-vscode-remote.remote-ssh-0.65.8\out\extension.js:1:526942
[21:47:19.695] ------
```

**Comment by yanhui09** [comment]
Updates:
I could use remote vscode-server with my macbook, but it still didn't work for my windows PC.

**Comment by yanhui09** [comment]
I made a typo before.
I **could** connet the remote host using ssh in powershell.

**Comment by tanhakabir** [comment]
Hmm you could try running the command `Kill VS Code Server on Host` before trying to connect?

@roblourens any other ideas for what could be happening here?

**Comment by yanhui09** [comment]
Yes. I have tried that before, it didn't work for me. 
The good news is that now I can connect to the remote server from my windows pc, after I used my macbook to connect to the remote server. (The connection from macbook made a new `.vscode-server` there, as I have removed the old one) 

I'm not sure but I think it's something stuck on my windows PC. It couldn't connect to the server with the old config, and couldn't reinstall the `vscode-server` after I removed it. 

Anyway, I can work with it now. Thanks for the help.

**Comment by ZhangGe6** [comment]
I met the same issue. I tried the methods suggested in this [issue](https://github.com/microsoft/vscode-remote-release/issues/2805) and reinstalling VS code, but neither of them works for me. Finally, I got the solution mentioned [here](https://issueexplorer.com/issue/microsoft/vscode-remote-release/5693), which works for me. My solution is shared here as a reference:
- I found my cmd.exe crashed (it would just pop up and close immediately), which may be the cause of my issue.
- I change the registry values `Autorun`  in `HKEY_CURRENT_USER\Software\Microsoft\Command Processor` from `if exists` to a blank (as this [link](https://superuser.com/questions/1493887/what-is-the-default-registry-value-for-command-processors-autorun) suggests).
- my cmd.exe works fine and Remote SSH works again.
